core/controllers/admin.py

```diff
@@ -1066,3 +1066,30 @@ def get(self):
             'number_of_pending_deletion_models': (
                 wipeout_service.get_number_of_pending_deletion_requests())
         })
+
+
+class GetModelsRelatedToUserHandler(base.BaseHandler):
+    """Handler for getting all the models related to a user with specific ID."""
```

Comment:
This handler only returns whether the model exists or not (i.e, a boolean value), right?